services/registry/docker-compose.aws.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.7'
+services:
+  registry:
+      image: registry:2
+      environment:
+        REGISTRY_STORAGE_REDIRECT_DISABLE: "true"
+        REGISTRY_STORAGE_CACHE_BLOBDESCRIPTOR: inmemory
+        REGISTRY_STORAGE_REDIRECT_DISABLE: "true"
+        REGISTRY_STORAGE_DELETE_ENABLED: "true"
+      networks:
+          - public
```

Comment:
also already in the base docker-compose

Reply:
these should be removed.
Also fix the version of the registry (currently I think it's 2.7.x)